site/public/js/rainbow-customization.js

```diff
@@ -271,6 +271,100 @@ function getGradeableBuckets() {
     return gradeables;
 }
 
+function getPlagiarism() {
+    const plagiarismData = [];
+
+    const tableBody = document.getElementById('table-body');
+    const rows = tableBody.getElementsByTagName('tr');
+
+    for (let i = 0; i < rows.length; i++) {
+        const row = rows[i];
+        const user = row.cells[0].innerHTML;
+        const gradeable = row.cells[1].innerHTML;
+        const penalty = parseFloat(row.cells[2].innerHTML);
+
+        plagiarismData.push({
+            user: user,
+            gradeable: gradeable,
+            penalty: penalty,
+        });
+    }
+
+    return plagiarismData;
+}
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
It would be better to put the name of this function in an `/** exported */` block at the top of the file as discussed [here](https://submitty.org/developer/software_and_system_design/coding_style_guide/javascript#style-rules).